### PR TITLE
RNs-4.10.25:adds zstream

### PR DESCRIPTION
**Pull request by JoeAldinger**
RNs-4.10.25: adds zstream update

Version(s):
4.10

Issue:

Link to docs preview:
[Preview](https://joealdinger.github.io/openshift-docs/RNs-4.10.25/release_notes/ocp-4-10-release-notes.html)

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3169,3 +3169,26 @@ link:https://access.redhat.com/solutions/6969074[{product-title} 4.10.24 contain
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-25"]
+=== RHSA-2022:5730 - {product-title} 4.10.24 bug fix and security update
+
+Issued: 2022-08-01
+
+{product-title} release 4.10.25 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5730[RHSA-2022:5730] advisory. TThe RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:5729[RHSA-2022:5729] advisory.
```
Comment (Wiharris) [inline release_notes/ocp-4-10-release-notes.adoc:3178]: release 4.10.25 is now available.
```suggestion
{product-title} release 4.10.25, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5730[RHSA-2022:5730] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:5729[RHSA-2022:5729] advisory.
```

**Review by Wiharris — COMMENTED** [review]
Left some tiny suggestions. Otherwise, LGTM.

**Comment by johnwilkins** [comment]
LGTM

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -3169,3 +3169,26 @@ link:https://access.redhat.com/solutions/6969074[{product-title} 4.10.24 contain
...
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6970049[{product-title} 4.10.25 container image list]
+
+[id="ocp-4-10-25-bug-fixes"]
+==== Bug fixes
+
+* Previously, where clusters had a Security Context Constraint (SCC), the default IngressController Deployment could cause pods to fail to start. This was due to the default container name `router` being created without requesting sufficient permissions in the `securityContext` of the container. With this update, router pods will be admitted to the correct SCC and created without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2079034[*BZ#2079034*])
+
+* Previously, routers in the terminating state delay the `oc cp` command, which delayed the `must- gathers`. With this update, a timeout for each `oc cp` command has been set eliminating the delay of the `must-gathers`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2106842[*BZ#2106842*])
```
Comment (johnwilkins) [inline release_notes/ocp-4-10-release-notes.adoc:3189]: There's a space between must-gathers

**Pull request merged**